### PR TITLE
Updating the message and button text of the popup message asking a person if they want to see the new user tour

### PR DESCRIPTION
**Pull request by eiffel777**
Updating the text of the pop up message when a person logs in that asks them if they want to view the User Tour.

## Tests performed
Tested in docker

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: html/gui/js/modules/NoviceUser.js [hunk 1]
```diff
@@ -136,8 +136,8 @@ Ext.extend(XDMoD.Module.Summary, XDMoD.PortalModule, {
                                     Ext.Msg.show({
                                         cls: 'new-user-tour-dialog-container',
                                         title: 'New User Tour',
-                                        msg: "Welcome to XDMoD. It seems that you haven't viewed our User Tour yet. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. If you want to view the tour now click Yes below.<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show me this again.",
-                                        buttons: Ext.Msg.YESNO,
+                                        msg: "Welcome to XDMoD. It seems that you haven't viewed our User Tour yet. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. Would you like to view the User Tour now?<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show this message again.",
+                                        buttons: {no: "No, don't show me the User Tour", yes: "Yes, Show me the User Tour"},
```
Comment (jpwhite4) [inline html/gui/js/modules/NoviceUser.js]: This is quite wordy. How about "Start Tour" and "Close" as the button labels?

File: html/gui/js/modules/NoviceUser.js [hunk 2]
```diff
@@ -136,8 +136,8 @@ Ext.extend(XDMoD.Module.Summary, XDMoD.PortalModule, {
                                     Ext.Msg.show({
                                         cls: 'new-user-tour-dialog-container',
                                         title: 'New User Tour',
-                                        msg: "Welcome to XDMoD. It seems that you haven't viewed our User Tour yet. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. If you want to view the tour now click Yes below.<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show me this again.",
-                                        buttons: Ext.Msg.YESNO,
+                                        msg: "Welcome to XDMoD. It seems that you haven't viewed our User Tour yet. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. Would you like to view the User Tour now?<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show this message again.",
```
Comment (jpwhite4) [inline html/gui/js/modules/NoviceUser.js]: **contractions**: Don't use 'em in technical writing [1]



[1] Young, M. _The Technical Writer's Handbook_. University Science Books 1989

Comment (jpwhite4) [reply html/gui/js/modules/NoviceUser.js]: In fact you could remove the sentence "It seems that you haven't viewed our User Tour yet." and the same information would be imparted to the user.

File: html/gui/js/modules/NoviceUser.js [hunk 3]
```diff
@@ -136,8 +136,8 @@ Ext.extend(XDMoD.Module.Summary, XDMoD.PortalModule, {
                                     Ext.Msg.show({
                                         cls: 'new-user-tour-dialog-container',
                                         title: 'New User Tour',
-                                        msg: "Welcome to XDMoD. It seems that you haven't viewed our User Tour yet. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. If you want to view the tour now click Yes below.<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show me this again.",
-                                        buttons: Ext.Msg.YESNO,
+                                        msg: "Welcome to XDMoD. The User Tour is a short series of information tips giving an overview of some basic components of XDMoD. Would you like to view the User Tour now?<br /><br /><input type='checkbox' id='new-user-tour-checkbox' /> Please don't show this message again.",
```
Comment (plessbd) [inline html/gui/js/modules/NoviceUser.js]: Personally I would make the checkbox

"Show tour at next login."

I have noticed the trend of things like this requiring confirmation to keep showing needing a positive action

Also, some text like "The tour can be started anytime from the help menu."

**Pull request merged**